File: imeureka/stack_queue/주식가격.js
function solution(prices) {
	let answer = [0, 0, 0, 0, 0];
	for (i = 0; i < prices.length; i++) {
		for (j = i + 1; j < prices.length; j++) {
			answer[i]++;
			if (prices[i] > prices[j]) {
				break;
			}
		}
	}
	return answer;
}

File: howooyeon/stack_queue/주식가격.java
class Solution {
  public int[] solution(int[] prices) {
      int[] answer = new int[prices.length];
          for(int i = 0; i < prices.length; i++){
              int count = 0;
              for(int j = i + 1; j < prices.length; j++){
                  count++;
                  if(prices[i] > prices[j]) break;
              }
              answer[i] = count;
          }
      return answer;
  }
}